hwilib/base58.py

```diff
@@ -87,23 +87,6 @@ def to_address(b, version):
     data += checksum
     return encode(data)
 
-def pubkey_to_address(pubkey, testnet=False):
-    pkh = hash160(pubkey)
-    if testnet:
-        return to_address(pkh, b'\x6f')
-    else:
-        return to_address(pkh, b'\x00')
-
-def xpub_to_address(xpub, testnet=False):
-    data = decode(xpub)
-    pubkey = data[-37:-4]
-    return pubkey_to_address(pubkey, testnet)
-
-def xpub_to_pub_hex(xpub):
```

Comment:
I still need `xpub_to_pub_hex` in #117, because the inferred descriptors from Core use the pubkey in hex.

Reply:
I've added it back in